Stripe/STPAPIClient.m

```diff
@@ -515,7 +515,9 @@ - (void)retrieveSourceWithId:(NSString *)identifier clientSecret:(NSString *)sec
     }];
 }
 
-- (NSURLSessionDataTask *)retrieveSourceWithId:(NSString *)identifier clientSecret:(NSString *)secret responseCompletion:(STPAPIResponseBlock)completion {
+- (NSURLSessionDataTask *)retrieveSourceWithId:(NSString *)identifier
+                                  clientSecret:(NSString *)secret
+                            responseCompletion:(void (^)(STPSource * _Nullable, NSHTTPURLResponse * _Nullable, NSError * _Nullable))completion {
```

Comment:
This completion block is passed straight through into `getWithAPIClient`, whose signature is expecting a different block structure.

See: https://github.com/stripe/stripe-ios/blob/47a13b20f155afe759563ee0c0b3161f6337026c/Stripe/STPAPIRequest.h#L50